src/session.ts

```diff
@@ -247,14 +249,24 @@ export class Session extends common.GrpcServiceObject {
         this.databaseRole =
           options.databaseRole || database.databaseRole || null;
 
-        return database.createSession(options, (err, session, apiResponse) => {
-          if (err) {
-            callback(err, null, apiResponse);
-            return;
-          }
+        const q = {opts: database.observabilityConfig};
+        return startTrace('Session.create', q, span => {
```

Comment:
I think we dont need this as well, as we dont use single session creation , instead we use batch create session